packages/shared_preferences/shared_preferences_android/AUTHORS

```diff
@@ -63,4 +63,4 @@ Juan Alvarez <juan.alvarez@resideo.com>
 Aleksandr Yurkovskiy <sanekyy@gmail.com>
 Anton Borries <mail@antonborri.es>
 Alex Li <google@alexv525.com>
-Rahul Raj <64.rahulraj@gmail.com>
+Rahul Raj <64.rahulraj@gmail.com>
```

Comment:
Please don't make whitespace-only changes to files (especially not to remove newlines).

Reply:
Done! :)